tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -35,8 +35,11 @@ describe(__filename, () => {
   });
 
   function render({ location, ...props } = {}) {
+    const _config = getFakeConfig({ enableFeatureUseUtmParams: false });
```

Comment:
If the plan is to turn the feature on, wouldn't it make more sense for the test to assume the feature is on, and not off? Are you just doing this to keep the patch small, and deferring updating the test to a different patch?

Reply:
Yeah, the plan is to remove the feature flag once this is live.